packages/toolkit/src/combineSlices.ts

```diff
@@ -121,7 +122,7 @@ export interface CombinedSliceReducer<
     config?: InjectConfig,
   ): CombinedSliceReducer<
     InitialState,
-    Id<DeclaredState & WithSlice<SliceLike<ReducerPath, String>>>
+    Id<DeclaredState & WithSlice<SliceLike<ReducerPath, string>>>
```

Comment:
```suggestion
    Id<DeclaredState & WithSlice<SliceLike<ReducerPath, State>>>
```
huh, apparently this is a bug i never noticed - should probably have its own PR